docs/mine-hnt/hotspot-makers/hotspot-ble-services/hotspot-ble-services.mdx

```diff
@@ -9,12 +9,26 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 
 # Hotspot BLE Services
 
-The Helium Hotspot provides a Bluetooth Low Energy\(BLE\) interface for users to configure several aspects of the device. The BLE services and advertisement data are provided below. You must first press the black button on left side of the Hotspot to activate the BLE advertising before attempting to connect.
+The Helium Hotspot provides a Bluetooth Low Energy\(BLE\) interface for users to
+configure several aspects of the device. The BLE services and advertisement data
+are provided below.
 
+It is generally recommended that the user first press a button on the Hotspot to
+activate the BLE advertising. Alternatively, enabling BLE for ~5 minutes after
+power up is acceptable. Maintaining BLE generally available is not
+recommended.to
```

Comment:
Maintaining BLE advertising for a prolonged period of time is not recommended.

Reply:
Thanks - fixed it